examples/ethers_base/test/tests.ts

```diff
@@ -4,179 +4,138 @@
-                    `${origin}/transfer-from-sepolia-faucet-wallet`,
-                    {
-                        method: 'POST',
-                        headers: [['Content-Type', 'application/json']],
-                        body: JSON.stringify({
-                            to: canisterAddress,
-                            value: '.0001'
-                        })
-                    }
-                );
-                const responseText = await response.text();
-
-                return {
-                    Ok: responseText.startsWith('transaction sent with hash:')
-                };
-            }
-        },
-        {
-            name: '/address-balance canister',
-            test: async () => {
-                const response = await fetch(`${origin}/address-balance`, {
-                    method: 'POST',
-                    headers: [['Content-Type', 'application/json']],
-                    body: JSON.stringify({
-                        address: canisterAddress
-                    })
-                });
-                const responseJson = await response.json();
-
-                return {
-                    Ok: BigInt(responseJson.__bigint__) === 100_000_000_000_000n
-                };
-            }
-        },
-        {
-            name: '/transfer-from-canister to caller wallet',
-            test: async () => {
-                const response = await fetch(
-                    `${origin}/transfer-from-canister`,
-                    {
-                        method: 'POST',
-                        headers: [['Content-Type', 'application/json']],
-                        body: JSON.stringify({
-                            to: callerAddress,
-                            value: '.000000000000000007'
-                        })
-                    }
-                );
-                const responseText = await response.text();
-
-                return {
-                    Ok: responseText.startsWith('transaction sent with hash:')
-                };
-            }
-        },
-        {
-            name: '/address-balance canister',
-            test: async () => {
-                const response = await fetch(`${origin}/address-balance`, {
-                    method: 'POST',
-                    headers: [['Content-Type', 'application/json']],
-                    body: JSON.stringify({
-                        address: canisterAddress
-                    })
-                });
-                const responseJson = await response.json();
-
-                return {
-                    Ok: BigInt(responseJson.__bigint__) < 100_000_000_000_000n
-                };
-            }
-        },
-        {
-            name: '/address-balance caller',
-            test: async () => {
-                const response = await fetch(`${origin}/address-balance`, {
+    return () => {
+        it("gets the caller's address using ethers and ThresholdWallet", async () => {
```

Comment:
We should change these to the correct HTTP methods and use the force query/update headers

Reply:
done